packages/faustwp-cli/package.json

```diff
@@ -9,21 +9,31 @@
     "faust": "dist/index.js"
   },
   "devDependencies": {
+    "@types/archiver": "^5.3.2",
     "@types/configstore": "^6.0.0",
     "@types/cookie": "0.5.1",
     "@types/dotenv-flow": "^3.2.0",
+    "@types/fs-extra": "^11.0.2",
+    "@types/isomorphic-fetch": "^0.0.36",
+    "@types/jest": "^29.5.5",
     "@types/node": "^18.15.11",
-    "@types/jest": "^27.0.2",
     "@types/prompt": "1.1.2",
     "@types/uuid": "8.3.4",
-    "rimraf": "^3.0.2",
-    "ts-jest": "^27.1.5",
+    "rimraf": "5.0.5",
+    "ts-jest": "^29.1.1",
     "typescript": "^4.9.5"
   },
+  "peerDependencies": {
+    "@wordpress/scripts": "^26.13.0"
+  },
```

Comment:
Hmm. this new peer dep. may be a breaking change 🤔 

For example, once this gets released, all users of `@faustwp/cli` will need `@wordpress/scripts` as a peer dependency, otherwise NPM will yell at them. However, not everyone is going to be using the `blockset` command. How should we account for this?

cc @theodesp

Reply:
Actually we cannot use this. I will remove it.